loader/base_loader.py

```diff
@@ -108,6 +118,106 @@ def __init__(self, dss_endpoint: str, staging_bucket: str, google_project_id: st
         dss_config['DSSClient'].swagger_url = f'{self.dss_endpoint}/swagger.json'
         self.dss_client = DSSClient(config=dss_config)
 
+    @staticmethod
+    def get_s3_metadata_client(aws_meta_cred, session='NIH-Test', duration=43199):
+        """
+        Access AWS credentials from a file and supply a client for them.
+
+        :param aws_meta_cred: File containing an AWS ARN for an AssumedRole, e.g.:
+                              'arn:aws:iam::************:role/ROLE_NAME_HERE'
+        :param duration: How long, in seconds, the AssumedRole will be valid for.
+        :return: An AWS s3 client object authorized with the above credentials or None.
+        """
+        if not aws_meta_cred:
+            return None
+
+        client = boto3.client('sts')
+        with open(aws_meta_cred, 'r') as f:
+            rolearn = f.read().strip()
+
+        # DurationSeconds can have a value from 900s to 43200s (as of 10.23.2018).
+        # 900s = 15 min; 43200s = 12 hours
+        # https://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html
+        print(rolearn)
```

Comment:
If you are going to output the role arn, please log it (e.g. at info level) rather than print it, and include a message indicating what the value is. If you want to just remove the print line, that's fine, too.